peps/pep-0790.rst

```diff
@@ -0,0 +1,469 @@
+PEP: 790
+Title: New environment markers for modern Python environments
+Author: Klaus Zimmermann <klaus_zimmermann@gmx.de>,
+        Ralf Gommers <ralf.gommers@gmail.com>
+Sponsor: Lysandros Nikolaou <lisandrosnik@gmail.com>
```

Comment:
@lysnikolaou can you confirm sponsorship?

Reply:
See https://github.com/python/peps/pull/4315#issuecomment-2743569002